package.json

```diff
@@ -35,6 +35,8 @@
     "postcss-reporter": "^4.0.0",
     "postcss-url": "^7.1.0",
     "react-hot-loader": "^1.3.1",
+    "request": "^2.81.0",
```

Comment:
Did you actually test out in the browser to ensure that you get the correct response type back? I'm not entirely sure these packages work outside of a node context.